apps/gp2-server/scripts/migrate-entity-outputs.ts

```diff
@@ -0,0 +1,73 @@
+import { getGraphQLClient as getContentfulGraphQLClient } from '@asap-hub/contentful';
+import {
+  contentfulAccessToken,
+  contentfulEnvId,
+  contentfulSpaceId,
+} from '../src/config';
+
+import { OutputContentfulDataProvider } from '../src/data-providers/output.data-provider';
+import { getContentfulRestClientFactory } from '../src/dependencies/clients.dependency';
+import { isInternalUser } from '@asap-hub/validation';
+
+console.log('Migrate related entities refs in outputs...');
+
+const contentfulGraphQLClient = getContentfulGraphQLClient({
+  space: contentfulSpaceId,
+  accessToken: contentfulAccessToken,
+  environment: contentfulEnvId,
+});
+
+const outputDataProvider = new OutputContentfulDataProvider(
```

Comment:
Not sure if this is a big deal, but I believe this only fetches published outputs right? In production it seems there are two drafts with related entity. Maybe it will be necessary to update it manually then

![Screenshot 2023-10-04 at 14 02 36](https://github.com/yldio/asap-hub/assets/16595804/44a4d8d0-8f33-434b-b444-c22fc611afe6)

Reply:
yes, but once they try to publish those, it will give them an error since the related entities field is required. and i don't know if they really want to show those draft entries...

Reply:
ah ok, I think it makes sense not to change the code 😉  I was just thinking about migrating the related entity to related entities manually without publishing it before we remove related entity completely so they don't lose track of what it was before